ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -301,11 +315,15 @@ class ReactIterableAPI: RCTEventEmitter {
         IterableAPI.updateUser(dataFields, mergeNestedObjects: mergeNestedObjects)
     }
     
-    @objc(updateEmail:)
-    func updateEmail(email: String) {
+    @objc(updateEmail:authToken:)
+    func updateEmail(email: String, with authToken: String?) {
```

Comment:
Is this function call supposed to have a with keyword? Seems unusual

Reply:
haha yes..  thats how iOS functions are written to make function names more understandable...